metricbeat/metricbeat.full.yml

```diff
@@ -68,8 +68,9 @@ metricbeat.modules:
   period: 10s
   processes: ['.*']
 
-  # if true, exports the CPU usage in ticks, together with the percentage values
-  #cpu_ticks: false
+  # Configure the metric types that are included by these metricsets.
+  cpu.metrics:  [percentages]  # The other available options are normalized_percentages and ticks.
```

Comment:
Should we add "quotes" around percentages? Seems we do this everywhere else with strings.